src/iadmin.cpp

```diff
@@ -2056,12 +2076,23 @@ usage( char *subOpt ) {
         ""
     };
 
-
-    char *moduserMsgs[] = {
-        " moduser Name[#Zone] [ type | comment | info | password ] newValue",
+    char* moduserMsgs[] = {
+        " moduser Name[#Zone] [ type | comment | info | password | remove_password ] newValue [no-scramble]",
         "Modifies a field of an existing user definition.",
+        " ",
         "For password authentication, use moduser to set the password.",
-        "(The password is transferred in a scrambled form to be more secure.)",
+        "By default, the password is transferred in a scrambled form to be more secure.",
+        "Use the no-scramble option after the newValue parameter when setting the password",
+        "to indicate that the password should not be scrambled before being sent to the",
+        "server. This option is required when connected to servers running in FIPS mode.",
```

Comment:
Is it "FIPS mode" or a "FIPS-compliant mode"?

Should we mention FIPS at all?
Do we need to come up with a name for FIPS related options?

Hmm...

Reply:
Sentence removed.